crates/infra/utils/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::missing_errors_doc)]
```

Comment:
I think we unconditionally allow htis?

Reply:
Good catch, it's a leftover when I tried to silence this for local-only crates!